rust/json-gen/src/main.rs

```diff
@@ -156,8 +155,5 @@ fn main() {
     gen_json_schema!(BigInt);
     gen_json_schema!(Int);
     gen_json_schema!(Value);
-
-    // tx_builder.rs
-    gen_json_schema!(PlutusWitness);
-    gen_json_schema!(PlutusWitnesses);
+    gen_json_schema!(TransactionUnspentOutput);
```

Comment:
@vsubhuman  I'm not sure that we should serialize TransactionUnspentOutput but there is cbor serialization for this type in serlib.